src/Shell/Task/PluginTask.php

```diff
@@ -235,7 +237,7 @@ protected function _modifyAutoloader($plugin, $path)
             $cwd = getcwd();
 
             // Windows makes running multiple commands at once hard.
-            chdir(dirname($path));
+            chdir(ROOT);
```

Comment:
Indeed, the changes was related to that issue. Regarding _rootComposerFilePath(), that would be  an unnecessary function call since the function is also just:
"return ROOT . DS . 'composer.json';"

However, if requested I can ofc change it to use the function call. dirname($file) is sufficient, since we do not want to step out of the root, hu?